store/mongo/datastore_mongo.go

```diff
@@ -474,6 +484,25 @@ func (db *DataStoreMongo) GetJobByNameAndID(
 	return &job, nil
 }
 
+// GetJobByNameAndID get the task execution status for a job
+// by workflow name and ID
+func (db *DataStoreMongo) GetAllJobs(
+	ctx context.Context) ([]model.Job, error) {
```

Comment:
As the number of jobs is going to grow (nothing removes old jobs), I believe it makes sense to sort (desc) and get the first N jobs, optionally skipping M jobs, effectively implementing pagination.

Reply:
added. I also added the time of the job was [inserted](https://github.com/mendersoftware/workflows/pull/39/commits/a2882aa80c56c446976cc0d5028bcaa4ce6a36f6#diff-e00004e5336b2d40b5eaa76e50e24e64R58).